Changes needed in server.cfg, mainly :
    Making use of RabbitMQ (default is ActiveMQ)
    The default port used to connect to RabbitMQ is 61613
    Default user : 'mcollective' & passwd : 'changeme'
    Change RabbitMQ to log to /dev/stderr, so it goes to the SMF log

NOTE : This patch is developed in-house (and Solaris specific)

--- mcollective-2.8.2/etc/server.cfg.dist.orig	2015-05-14 08:50:32.000000000 -0700
+++ mcollective-2.8.2/etc/server.cfg.dist	2016-04-08 12:04:13.523097348 -0700
@@ -1,7 +1,9 @@
 main_collective = mcollective
 collectives = mcollective
-libdir = /usr/libexec/mcollective
-logfile = /var/log/mcollective.log
+libdir = /usr/share/mcollective
+logger_type = file
+# Send output to SMF service log
+logfile = /dev/stderr
 loglevel = info
 daemonize = 1
 
@@ -9,13 +11,14 @@
 securityprovider = psk
 plugin.psk = unset
 
-connector = activemq
-plugin.activemq.pool.size = 1
-plugin.activemq.pool.1.host = stomp1
-plugin.activemq.pool.1.port = 6163
-plugin.activemq.pool.1.user = mcollective
-plugin.activemq.pool.1.password = marionette
+connector = rabbitmq
+plugin.rabbitmq.vhost = /mco
+plugin.rabbitmq.pool.size = 1
+plugin.rabbitmq.pool.1.host = localhost
+plugin.rabbitmq.pool.1.port = 61613
+plugin.rabbitmq.pool.1.user = mcollective
+plugin.rabbitmq.pool.1.password = changeme
 
 # Facts
 factsource = yaml
 plugin.yaml = /etc/mcollective/facts.yaml
